src/WebGPU/pointer.ts

```diff
@@ -179,6 +177,27 @@ export default function initMouseController(
         event.preventDefault()
         Logic.commitChanges()
         break
+      case '=':
+      case '+':
+        // Zoom in with Ctrl/Cmd + Plus
+        if ((!event.ctrlKey && !event.metaKey)) break
+        event.preventDefault()
+        {
+          const centerX = pointer.x !== OUTSIDE_CANVAS ? pointer.x : canvas.width / 2
+          const centerY = pointer.y !== OUTSIDE_CANVAS ? pointer.y : canvas.height / 2
+          performZoom(0.1, centerX, centerY)
+        }
+        break
+      case '-':
```

Comment:
What if shift is pressed? Will is still work or it need to also handle underscore character(its a minus while shift is pressed)?